handbook/openid-connect/openid-connect.md

```diff
@@ -316,3 +316,17 @@ The following sources can be used to troubleshoot a failed connection.
     SystemLink uses log rotation therefore the latest logs may be in one of the numbered `error.log` files.
 
 **Returned Claims:** See [**Viewing Claims Returned by a Provider**](#viewing-claims-returned-by-a-provider).
+
+### Cache Entry Size
+
+The OpenID Connect module stores information in a shared memory cache. If a cache entry is too large, users will see an "Internal Server Error" when attempting to log in, and the NI Web Server logs will contain the following entries
+```
+    oidc_cache_shm_set: could not store value since value size is too large
+    oidc_cache_set: could NOT store X bytes in shm cache backend for key Y
+```
```

Comment:
Until we get it added to the build workflow, please open the Markdown file in VS Code with the markdownlint extension and resolve any warnings:

- There should be blank lines before and after code snippets.
- You should specify the language for the code snippet (text).
- For consistency, we should wrap it in an admonition.

How about something like:

```md
The OpenID Connect module stores information in a shared memory cache. If a cache entry is too large, users will see an **Internal Server Error** when attempting to log in.

!!! note "Example Error Logs"
    When this happens, the NI Web Server error logs will contain entries like the following:

    ```text
    oidc_cache_shm_set: could not store value since value size is too large
    oidc_cache_set: could NOT store X bytes in shm cache backend for key Y
    ```
```